### PR TITLE
Spnego support java 11 new messages

### PR DESCRIPTION
**Pull request by utle**

### CODE REVIEW

**Comment by utle** [comment]
#5462

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

File: dev/com.ibm.ws.security.jaas.common/resources/com/ibm/ws/security/jaas/common/internal/resources/JAASCommonMessages.nlsprops [hunk 1]
```diff
@@ -66,4 +66,16 @@ JAAS_CUSTOM_LOGIN_MODULE_CLASS_NOT_FOUND.useraction=Verify that all custom JAAS
...
 JAAS_DUPLICATE_ENTRY_NAME.explanation=A Duplicate login configuration name was specified in the JAAS configuration file and server.xml/client.xml file
-JAAS_DUPLICATE_ENTRY_NAME.useraction=Check the JAAS configuration file and server.xml/client.xml file.
\ No newline at end of file
+JAAS_DUPLICATE_ENTRY_NAME.useraction=Check the JAAS configuration file and server.xml/client.xml file.
+
+JAAS_KRB5_LOGIN_MODULE_PACKAGE_ERROR=CWWKS1143E: The {0} login module is not supported for version {1} by the {2} Java vendor.
+JAAS_KRB5_LOGIN_MODULE_PACKAGE_ERROR.explanation=The Krb5LoginModule module specified in the server.xml file, the jaas.conf file, or both files is not supported in the Java vendor and version that was used.
+JAAS_KRB5_LOGIN_MODULE_PACKAGE_ERROR.useraction=Use the correct Java vendor and version that is supported for the Krb5LoginModule module or use a Krb5LoginModule module that is supported by the Java vendor and version.
+
+JAAS_KRB5_LOGIN_MODULE_INVALID_OPTIONS=CWWKS1144E: The {0} login module options are not supported with the {1} Java vendor and the version {2}.
```
Comment (chirp1) [inline dev/com.ibm.ws.security.jaas.common/resources/com/ibm/ws/security/jaas/common/internal/resources/JAASCommonMessages.nlsprops]: Change "and the version" to "and version".

File: dev/com.ibm.ws.security/resources/com/ibm/ws/security/internal/resources/LoggingMessages.nlsprops [hunk 2]
```diff
@@ -53,3 +53,7 @@ SECURITY_SERVICE_MULTIPLE_SERVICE_AVAILABLE.useraction=Specify only one service
 SEC_TOO_MANY_PRINCIPALS=CWWKS0010E: While getting the caller principal, the caller subject was found to have more than one principal of type WSPrincipal. Only one WSPrincipal can exist in the subject. The names of the WSPrincipals are: {0}
 SEC_TOO_MANY_PRINCIPALS.explanation=The caller's subject has too many principals of type WSPrincipal. Only one principal of this type is supported.
 SEC_TOO_MANY_PRINCIPALS.useraction=Review the application to understand why the subject has more than one WSPrincipal.
+
+SPNEGO_API_IS_NOT_SUPPORT=CWWKS0011E: The {0} SPNEGO API is not supported for the version {0} of the {1} Java runtime.
```
Comment (chirp1) [inline dev/com.ibm.ws.security/resources/com/ibm/ws/security/internal/resources/LoggingMessages.nlsprops]: Change "for the version" to "for version".

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
The message change should not cause these errors in the personal build

**Pull request merged**